samples/sample_hevc_fei/include/sample_hevc_fei_defs.h

```diff
@@ -30,6 +30,8 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 #define CHECK_STS_AND_RETURN(X, MSG, RET) {if ((X) < MFX_ERR_NONE) {MSDK_PRINT_RET_MSG(X, MSG); return RET;}}
 
+#define MSDK_ZERO_ARRAY(VAR, NUM) {memset(VAR, 0, sizeof(VAR[0])*NUM);}
```

Comment:
Please use template Zero

Reply:
Did you mean that I should use MSDK_ZERO_MEMORY to replace MSDK_ZERO_ARRAY? There are many MSDK_ZERO_MEMORY called in this file.

Reply:
no, I think Olga meant template function Zero, but I don't see any sense in using it. I also don't like such macro (I know that we used them in previous samples, but that isn't really good way to do zeroing data). I suggest to use std::fill for initialization and std::copy for copying and not to use macro